core/connector/src/main/java/org/exoplatform/wcm/connector/collaboration/OpenInOfficeConnector.java

```diff
@@ -132,7 +136,9 @@ public Response updateDocumentTitle(
       if (linkManager.isLink(node)) node = linkManager.getTarget(node);
       nodePath = node.getPath();
       isFile = node.isNodeType(NodetypeConstant.NT_FILE);
-    }catch(RepositoryException ex){
+    } catch (PathNotFoundException | AccessDeniedException e) {
+      return Response.status(Status.NOT_FOUND).build();
```

Comment:
Shouldn't we log something in this case ?

Reply:
Nope, not needed, we will not have to log any request that returns a regular and normal 404 HTTP code. (We have frontend access log for that)

Reply:
ok, will add debug log for that in case Ops wants to see entries of 404 here instead of access log of frontend

Reply:
done